src/scripts/btma/annotator.py

```diff
@@ -63,81 +65,101 @@ def main():
-        elif belonging == module[1]:
-            csv_table_per_module[1].append(['OEO:' + ids, ontology_iri + '/' + belonging])
-        elif belonging == module[2]:
-            csv_table_per_module[2].append(['OEO:' + ids, ontology_iri + '/' + belonging])
-        else:
-            csv_table_per_module[3].append(['OEO:' + ids, ontology_iri + '/' + belonging])
+        belongs_to_module = list(zip(ids, belonging))
+
+        csv_table = [["ID","belongs to module"],
+                     ["ID","AI OEO:belongs_to_module"]]
+
+        csv_table_per_module = (csv_table, csv_table, csv_table, csv_table)
+
+        for (ids, belonging) in belongs_to_module:
+            if belonging == module[0]:
+                csv_table_per_module[0].append(['OEO:' + ids, ontology_iri + '/' + belonging])
+            elif belonging == module[1]:
+                csv_table_per_module[1].append(['OEO:' + ids, ontology_iri + '/' + belonging])
+            elif belonging == module[2]:
+                csv_table_per_module[2].append(['OEO:' + ids, ontology_iri + '/' + belonging])
+            else:
+                csv_table_per_module[3].append(['OEO:' + ids, ontology_iri + '/' + belonging])
         
-    print('Creating ontologies...')
+        print('Creating ontologies...')
 
-    for (i, csv_table) in enumerate(csv_table_per_module):
-        f = open(script_path / 'btma' / "template.csv", 'w', newline='')
+        for (i, csv_table) in enumerate(csv_table_per_module):
+            f = open(script_path / 'btma' / "template.csv", 'w', newline='')
 
-        writer = csv.writer(f)
+            writer = csv.writer(f)
 
-        writer.writerows(csv_table)
+            writer.writerows(csv_table)
 
-        f.close()
+            f.close()
 
-        # Create OWL/XML with help of the CSV template
+            # Create OWL/XML with help of the CSV template
     
-        sp.call('java -jar {jar} template --template {template} \
-                                          --prefix "OEO: {iri}_" \
-                                          --output {out}'.format(jar=robot_path,
-                                                                 template=script_path / 'btma' / 'template.csv',
-                                                                 iri=ontology_iri,
-                                                                 out=csv_path / 'belongs-to-{m}-annotation.omn'.format(m=modules[i])))
+            sp.call('java -jar {jar} template --template {template} \
+                                              --prefix "OEO: {iri}_" \
+                                              --output {out}'.format(jar=robot_path,
+                                                                     template=script_path / 'btma' / 'template.csv',
+                                                                     iri=ontology_iri,
+                                                                     out=exp_path / 'belongs-to-{m}-annotation.omn'.format(m=modules[i])))
```

Comment:
Again I had to add `,shell=True` to the sp.call -- this is probably operating system dependent :-/

Reply:
Done